src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -74,42 +74,37 @@ public void RawSendInCaseOfFailure(IModel channel, string address, byte[] body,
 
         public void Initialize(IModel channel, string mainQueue)
         {
-
             CreateExchange(channel, mainQueue);
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
-
         }
 
-        static string ExchangeName(Type type)
-        {
-            return type.Namespace + ":" + type.Name;
-        }
+        static string ExchangeName(Type type) => type.Namespace + ":" + type.Name;
```

Comment:
> . In moving to C# 6, in .NET 4.6 we can simply swap all our current concatenations and invariant string.Formats for bling strings wrapped in Invariant()

Is this a more general statement? I don't recall seeing any `String.Format` with specified cultures anywhere in our codebases.